modules/alloy/src/test/ts/atomic/alien/BoxesTest.ts

```diff
@@ -0,0 +1,57 @@
+import { context, describe, it } from '@ephox/bedrock-client';
+import { assert } from 'chai';
+import fc from 'fast-check';
+
+import * as Boxes from 'ephox/alloy/alien/Boxes';
+
+// TODO: Add more tests.
```

Comment:
Should this have a ticket?

Reply:
Probably. Just because it was so far away from merging, I didn't want to create a ticket that we might never need (if it gets rejected, for example). It's asking to write tests for a function that doesn't exist yet, so it's contingent on a merge.

Reply:
I think I added that before putting in the property tests as well, so I'll just remove it for now.